scripts/elastic-certs.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+docker run --rm -v $(pwd)/certs:/certs docker.elastic.co/elasticsearch/elasticsearch:8.11.3 \
```

Comment:
move the docker version to a variable and ask the user for it interactively (this is supposed to be run by the user, right?)